tests/unit/backend/wmg/api/test_v1.py

```diff
@@ -776,6 +777,65 @@ def test__query_request_with_filter_dims__returns_valid_filter_dims(
                 self.assertEqual(dev_stage_terms_eth_2_dev_2, dev_stage_terms_eth_2_no_dev_filter)
                 self.assertNotEqual(eth_stage_terms_eth_2_dev_2, ethnicity_terms_eth_2_no_dev_filter)
 
+            with self.subTest("Additional queries are not performed when the secondary dimensions are not set"):
+                with patch("backend.wmg.api.v1.find_dim_option_values") as mock_dims:
+                    mock_dims.side_effect = find_dim_option_values
```

Comment:
nit: I think you can use `patch("...", wraps=find_dim_option_values)` to achieve this pass-through, which might be 0.5% clearer that we're using this mock only to "spy" on the call count. I haven't used this, but interwebs suggest as much.